### PR TITLE
Refactoring pages

### PR DESCRIPTION
**Pull request by Sharmaryan**
changed colors, fixed bugs in product page, fixed path to home page in login page

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,7 +1,7 @@
 :root {
   --font-medium: 1.4rem;
   --primary-color: white;
-  --secondary-color: black;
+  --secondary-color: royalblue;
```
Comment (NeerajVodala) [inline style.css]: try to use HEX or rgba(cause different browsers interpret same color differently). Rest everything looks good.
All the best.

**Review by NeerajVodala — APPROVED** [review]

File: style.css (hunk 1 above)
Comment (Sharmaryan) [reply style.css]: sure will change it

Comment (Sharmaryan) [reply style.css]: done

**Review by Rockyraven — APPROVED** [review]
code is clean and all good to me 
you can merge

**Pull request merged**